var/spack/repos/builtin/packages/rose/package.py

```diff
@@ -92,8 +92,9 @@ def patch(self):
                 f.write('992c21ad06893bc1e9e7688afe0562eee0fda021 1460595442')
 
     def autoreconf(self, spec, prefix):
-        bash = which('bash')
-        bash('build')
+        if not spec.satisfies('@develop'):
```

Comment:
I don't think this will work because the ROSE tarballs Spack downloads is just a tar of the Git repository, not a `make dist` of ROSE, which would generate a `configure` script. Therefore, the `build` command is required for every version currently.